src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -35,6 +35,44 @@ public override SarifLog VisitSarifLog(SarifLog v2SarifLog)
             return null;
         }
 
+        internal AnnotatedCodeLocationVersionOne CreateAnnotatedCodeLocation(Location v2Location)
+        {
+            AnnotatedCodeLocationVersionOne annotatedCodeLocation = null;
+
+            if (v2Location != null)
+            {
+                annotatedCodeLocation = new AnnotatedCodeLocationVersionOne
+                {
+                    Annotations = v2Location.Annotations?.Select(CreateAnnotation).ToList(),
+                    LogicalLocationKey = v2Location.FullyQualifiedLogicalName,
```

Comment:
You also have to fill in v1 FQLN. `LogicalLocationKey` is only allowed in FQLN is present.

Reply:
Fixed